src/sdk/pynni/nni/_graph_utils.py

```diff
@@ -510,6 +518,65 @@ def _build_index(self, nodes_op):
                 output_to_node[output] = node
         return name_to_node, input_to_node, output_to_node
 
+    def _is_key_func(self, node_cpp):
+        """
+        Judge if a cpp node is a key function node.
+        If so, we should not merge this node into the
+        adjacent node.
+        """
+        if node_cpp.kind().startswith('aten::'):
+            # the nodes that start with 'aten' are key function
+            # nodes
+            return True
+        if node_cpp.kind() in [LIST_UNPACK_KIND, TUPLE_UNPACK_KIND]:
+            # We cannot merge the List/Tuple
+            # Construct/Unpack func into other nodes, else it
+            # may lead to a graph construction error.
```

Comment:
what is this error like?

Reply:
Take the shufflenet as an example:
https://github.com/microsoft/nni/issues/2581